app/src/main/java/org/piwigo/helper/AccountHelper.java

```diff
@@ -30,14 +32,20 @@
 import org.piwigo.io.model.LoginResponse;
 import org.piwigo.ui.model.User;
 
-import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Observable;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+/* TODO: cleanup this class. It has strange methcods... also the Userlist could be improved
```

Comment:
Associated issue number for a TODO would be a plus

Reply:
May be a good idea, yes.